src/services/stripe-service/Dockerfile

```
@@ -0,0 +1,14 @@
+FROM base AS builder
+
+RUN pnpm build --filter=@repo/stripe-service
+
+FROM golang:1.25-trixie AS runner
+
+ENV NODE_ENV="production"
+ENV STRIPE_SERVICE_PORT="50054"
+
+COPY --from=builder /app/src/services/stripe-service/build/app .
```

Hadolint (2.14.0)

[warning] `COPY` to a relative destination without `WORKDIR` set.

(DL3045)